src/components/nature-of-business/NatureOfBusiness.vue

```diff
@@ -180,24 +184,16 @@ export default class NatureOfBusiness extends Vue {
 
   /** Whether we have stored NAICS data. */
   get haveNaics (): boolean {
-    return (!!this.naicsCode && !!this.naicsDescription)
+    return (!!this.naicsCode || !!this.naicsDescription)
```

Comment:
This is to handle the situation where we have a code and no description (or vice-versa).

(Yes, this is now a valid scenario.)